index.html

```diff
@@ -0,0 +1,154 @@
+            class="nav-icon material-icons menu" id="menu"
+          >
+            menu
+          </span>
+        </nav>
+        <div class="logo">
+          <img src="./Img/Svg/sony-logo-3066.png" alt="" />
+        </div>
+        <nav class="desktop-version" id="show">
+          <ul class="nav-bar">
+            <span
+              class="visible close" id="close"
+            >
+              <img src="./Img/Cl/close_FILL0_wght400_GRAD0_opsz48.png" alt="close" />
+            </span>
+            <li> <a class="navbar-fun" href="./about.html">About</a></li>
+            <li> <a class="navbar-fun" href="#Main-Program">Program</a> </li>
+            <li> <a class="navbar-fun" href="#Featured">Featured Games</a> </li>
+            <li> <a class="navbar-fun" href="#Studio">Studio</a> </li>
+            <li> <a class="navbar-fun" href="https://www.youtube.com/watch?v=T5VtfD2i3rc">Watch Live</a></li>
+          </ul>
+        </nav>
+      </header>
+      <section class="Home-1">
+        <div class="Home-Back">
+          <h2 class="Home-1-h2">"Hello! Playstation world"</h2>
+          <h1>
+            PLAYSTATION & Insomniac Games<br />
+            GAMES SHOWCASE<br />
+            2022
+          </h1>
+          <p>
+            Play brand new games the same day they release and hundreds of
+            additional high-quality games with Playstation Game Pass. Join us as
+            we showcase even more additions to your game pass catalog.
+          </p>
+          <h2>2021.10.27(TUE) ~ 28(WED)</h2>
+          <p>@ Redmond, Washington, US, Microsoft Headquarters</p>
+        </div>
+      </section>
+    </div>
+    <section class="main-page" id="Main-Program">
+      <h1 class="main-page-1">Main Program</h1>
+      <div class="border"></div>
+      <!-- Card 1 starting -->
+      <div class="card margin-l">
+        <div class="card-1 left">
+          <img src="./Img/Card-Icon/launch.png" alt="icon of rocket" />
+          <h5>launch</h5>
+          <p>
+            Show opener introductions with Erin Ashely and Malik. Secial look at
+            fans celebration of the xbox games showcase
+          </p>
+        </div>
+        <!-- Card -1 end -->
+        <!-- Card 2 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/playstation-png.png"
+            alt="icon of Playstation"
+          />
+          <h5>Playstation</h5>
+          <p>
+            Console exclusive games showcase. This includes PS5 new version
+            console exclusive trailer.
+          </p>
+        </div>
+        <!-- Card 2 end -->
+        <!-- Card 3 starting -->
+        <div class="card-1 left card-im">
+          <img
+            src="./Img/Card-Icon/ps-controler.png"
+            alt="icon of Playstation controler"
+          />
+          <h5>Controler</h5>
+          <p>
+            Console exclusive games showcase. This includes new Controler 
+            exclusive trailer.
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/Hassaanjbaig-code/Capstone-Project/pull/1#discussion_r1031361658 Please add the hover effct to your cards you can do some thing like 
```
card-1.left.card-im:hover {
  border: 1px solid white;
}
```